### ISSUE TITLE
Asking for wsdl v15 of ProcessShipmentRequest

### ISSUE THREAD

**Issue by ikks** [gtaylor/python-fedex#37]
Hello, we are creating international shipment labels, but it seems that the wsdl V13 is not supported in the new version of the API.

According to the docs we should be filling something like

    RequestedShipment.CustomsClearanceDetail.CommercialInvoice.TermsOfSale

But in the library we have V13 and not the V15 they expect we to use, do you know how we can get the new wsdl file?  Thank you for any pointer

P.D: we already sent a request to websupport@fedex.com , and did not find anything.

**Comment by gtaylor** [comment]
I haven't actively used python-fedex in quite some time, but there used to be a separate developer section of their website that had the WSDLs. I'd accept a pull request that incorporated the new WSDLs, if this is something you'd like to head up.

**Comment by ikks** [comment]
Thank you, I tried to use the V15 but I was unable to make it work, unfortunately suds was not giving errors regarding the validation of the data entered, the debug went a nightmare.

**Comment by gtaylor** [comment]
I've never really figured out how to improve the debugging process with suds. This has been a frustration since 2008 when this library came to be.

**Comment by ikks** [comment]
I followed the samples provided by Fedex and used exactly the same fields but somehow suds refused to validate the information according to the new wsdl, so I ended up creating by hand some dictionaries and serialized the xml so I could be able to make the SOAP call directly.  Not proud of it, but I didn't get a proper way to consume the webservice with suds.  Thank you for the pointer of the WSDL, Fedex also offers an XSD so it's possible to validate locally the generated SOAP call, instead of hammering the Fedex beta servers.  Here a gist of it https://gist.github.com/ikks/11f39cc3554a5c5ca5ff
